web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/GetCertificatesAvailableFunctionsImpl.java

```diff
@@ -87,14 +88,25 @@
             )
         );
 
-        resourceLinks.add(
-            ResourceLinkDTO.create(
-                ResourceLinkTypeDTO.SIGN_CERTIFICATE,
-                "Signera",
-                "Signerar intygsutkast",
-                true
-            )
-        );
+        if (certificate.getMetadata().getType().equalsIgnoreCase(Af00213EntryPoint.MODULE_ID)) {
```

Comment:
There is a feature to check if there will be sign and send. Consider to use it instead: authoritiesHelper
            .isFeatureActive(AuthoritiesConstants.FEATURE_SIGNERA_SKICKA_DIREKT, certificate.getMetadata().getType());
            
Also see if you can verify this with a unit test.